src/assets/sass/variables/colors.scss

```
@@ -9,7 +9,9 @@ $yellow-700: #b5aa30;
 $yellow-800: #8c8325;
 $yellow-900: #6b641c;
 
+$light-blue-25: #f4f8fb;
```

Stylelint (17.4.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)